modules/nvidia_plugin/src/transformer/cuda_graph_transformer.cpp

```diff
@@ -2,12 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "openvino/cc/ngraph/itt.hpp"
+#include "openvino/cc/pass/itt.hpp"
 #include "cuda_graph_transformer.hpp"
 
 #include <fmt/format.h>
 
+#include "openvino/openvino.hpp"
```

Comment:
I propose to avoid global headers in the plugin

Reply:
> I propose to avoid global headers in the plugin

Removed